facebook/facebook.c

```diff
@@ -421,6 +434,8 @@ fb_cb_api_messages(FbApi *api, GSList *msgs, gpointer data)
         if (gc != NULL) {
             if (mark) {
                 fb_api_read(api, msg->tid, TRUE);
+            } else {
+                gc->data = GINT_TO_POINTER(TRUE);
```

Comment:
Shouldn't this be using one of the `FB_UTIL_PTRBIT_*` macros to prevent from overwriting other potential bits?